src/org/zaproxy/zap/extension/websocket/WebSocketProxy.java

```diff
@@ -72,6 +73,11 @@
 	private static Comparator<WebSocketObserver> observersComparator;
 	
 	/**
+	 * Used to determine when to call which {@link WebSocketSenderListener}.
```

Comment:
Changed as follows:
<pre>Used to determine the order to call each WebSocketSenderListener.</pre>

I also changed the commend of `observersComparator`.